evergreen/index.html

```diff
@@ -139,6 +140,14 @@ <h2>Contact Us</h2>
 
     <section id="plant-chat">
         <h2>Plant Chat</h2>
+        <h3>RULES OF ENGAGEMENT</h3>
+        <h4> - Must be logged in to post</h4>
+        <h4> - Only png, jpg, and gif formats supported</h4>
+        <h4> - All image content must be <= 8MB</h4>
+        <h4> - All plant "glow-up" photos must include a banana for scale</h4>
```

Comment:
Rules of engagement? What are we in wartime now?

I'd just use an unordered list instead of abusing headings, throw on some CSS if you'd like

Reply:
a little cheeky on my part, I can remove it if you hate it tho

Reply:
CSS can be honestly added later

Reply:
I'll make some ul as suggested

Reply:
git pull